docs/MinimallyImpactfulServicing.md

```diff
@@ -28,6 +28,15 @@ deserialize its state. The net effect is that this part of the code can be
 reloaded without any user visible change. This part is referred to as eBPF Core
 or the core driver.
 
+To ensure that the state stored in eBPF Core remains consistent across updates,
+the following actions are taken:
+1. Calls from user mode are blocked in eBPF Proxy.
+2. eBPF programs are detached from attach points prior unloading the old code
+and reattached after loading the new code.
+
+The net effect is that events that would have triggered execution of eBPF
+programs during servicing will be dropped.
```

Comment:
This is not correct if you just do what lines 33-35 say.  Removing a BPF program that would drop bad packets will let them through, potentially bug checking the system if the BPF program blocks a 0-day.  Hence this proposal would result in a non-graceful reboot.

Reply:
Update the document to indicate that each hook provider is free to implement type specific handling of events during servicing (i.e. drop, queue, or block events).